server/api/inspect-url.post.ts

```diff
@@ -31,11 +31,11 @@ export default defineEventHandler(async (event) => {
   }
 
   const startTime = Date.now()
-  // TODO(serhalp) `$fetch` automatically throws on 4xx, but we'd like to treat those as valid.
   const { status, headers } = await $fetch.raw(normalizedUrl, {
```

Comment:
let's leave a comment here explaining why we're using raw and ignoreResponseError